backend/.erlang.mk/deps.log

```diff
@@ -1 +1,2 @@
 /Users/tylercalabrese/Dev/cs21-platformer/backend/deps/cowboy
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/jsx
```

Comment:
Is this like something that we're all gonna need to have our own local version of the path to it when we're running or what?

Reply:
oops i should add logfiles to the gitignore lol

Dependencies install on their own when you run `make` or `make run`